### PR TITLE
PIMOB-1410: Prebuilt UI scaffold was created.

### PR DESCRIPTION
**Pull request by maxim-nosov-cko**
## Issue

[PIMOB-1410: Create shell default UI on the sample app that we can add the prebuilt UI into](https://checkout.atlassian.net/browse/PIMOB-1410)

## Proposed changes

Created scaffold for:
- PaymentDetails screen
- PaymentForm screen
- Navigation for both SDK and test app

The default light style for the card number component was added.
The test app was refactored and payment form usage was added.

Top-level diagram was created [Frames SDK Screen/Components relation](https://checkout.atlassian.net/wiki/spaces/FS/pages/5397283283/Frames+SDK+Screen+Components+relation).

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

* [x] Reviewers assigned
* [x] I have performed a self-review of my code and manual testing
* [x] Lint and unit tests pass locally with my changes
* [ ] I have added tests that prove my fix is effective or that my feature works
* [ ] I have added necessary documentation (if applicable)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you choose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: example_app_frames/src/main/java/com/checkout/example/frames/ui/screen/MainActivity.kt [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        super.onCreate(savedInstanceState)
+        setContent { Navigator(this.applicationContext) }
+    }
+}
+
+@Composable
+fun Navigator(context: Context) {
+    val navController = rememberNavController()
+    val defaultPaymentFormConfig = PaymentFormConfig(
+        publicKey = "",
```
Comment (deepesh-vasthimal-cko) [inline example_app_frames/src/main/java/com/checkout/example/frames/ui/screen/MainActivity.kt:28]: You can inject the existing test key if you like which is used in existing test sample app code.

**Review by deepesh-vasthimal-cko — APPROVED** [review]

File: example_app_frames/src/main/java/com/checkout/example/frames/navigation/Screen.kt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package com.checkout.example.frames.navigation
+
+sealed class Screen(val route: String) {
+    object Home : Screen("ckoHomeScreen")
+
+    object DefaultUI : Screen("ckoDefaultUI")
```
Comment (chintan-soni-cko) [inline example_app_frames/src/main/java/com/checkout/example/frames/navigation/Screen.kt:6]: Can we put these screen identifiers into strings.xml files?

File: frames/src/main/java/com/checkout/frames/screen/navigation/Screen.kt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package com.checkout.frames.screen.navigation
+
+internal sealed class Screen(val route: String) {
+    object PaymentDetails : Screen("payment_details")
```
Comment (chintan-soni-cko) [inline frames/src/main/java/com/checkout/frames/screen/navigation/Screen.kt:4]: Can we put it into strings.xml file?

File: frames/src/main/java/com/checkout/frames/screen/paymentdetails/PaymentDetailsScreen.kt [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+) {
+    val viewModel: PaymentDetailsViewModel = viewModel(
+        factory = PaymentDetailsViewModel.Factory(injector, style)
+    )
+
+    Column(modifier = Modifier.padding(PaymentDetailsScreenConstants.padding.dp)) {
+
+        // TODO: Replace with header component
+        Text(
+            text = "Payment details",
```
Comment (chintan-soni-cko) [inline frames/src/main/java/com/checkout/frames/screen/paymentdetails/PaymentDetailsScreen.kt:32]: Can we put it into strings.xml file?

**Review by chintan-soni-cko — APPROVED** [review]
Few comments, except looks good to me

File: example_app_frames/src/main/java/com/checkout/example/frames/navigation/Screen.kt (hunk 2 above)
Comment (maxim-nosov-cko) [reply example_app_frames/src/main/java/com/checkout/example/frames/navigation/Screen.kt:6]: I will put them in constants.

File: frames/src/main/java/com/checkout/frames/screen/navigation/Screen.kt (hunk 3 above)
Comment (maxim-nosov-cko) [reply frames/src/main/java/com/checkout/frames/screen/navigation/Screen.kt:4]: Duplicate.

File: frames/src/main/java/com/checkout/frames/screen/paymentdetails/PaymentDetailsScreen.kt (hunk 4 above)
Comment (maxim-nosov-cko) [reply frames/src/main/java/com/checkout/frames/screen/paymentdetails/PaymentDetailsScreen.kt:32]: It's a dummy header, new header will be done appropriately in another ticket.

**Review by chintan-soni-cko — APPROVED** [review]

**Pull request merged**